src/main/java/frc/team2412/robot/commands/index/IndexShootCommand.java

```diff
@@ -20,12 +20,8 @@ public void initialize() {
 
     @Override
     public boolean isFinished() {
-        // stop motors when there are no balls left
-        if (!subsystem.ingestSensorHasBallIn() && !subsystem.feederSensorHasBallIn()) {
-            subsystem.ingestMotorStop();
-            subsystem.feederMotorStop();
-            return true;
-        }
+        subsystem.ingestMotorStop();
+        subsystem.feederMotorStop();
```

Comment:
?? This should immediately stop the motors but keep the command running, which seems odd.  The previous code seemed more correct:
```
        // stop motors when there are no balls left
        if (!subsystem.ingestSensorHasBallIn() && !subsystem.feederSensorHasBallIn()) {
            subsystem.ingestMotorStop();
            subsystem.feederMotorStop();
            return true;
        }
```